tools/backport_pr.py

```diff
@@ -2,20 +2,27 @@
 """
 Backport pull requests to a particular branch.
 
-Usage: backport_pr.py branch [PR] [PR2]
+Usage: backport_pr.py [org/repository] branch [PR] [PR2]
```

Comment:
> syntax if you've set up your git correctly

heh, that's a matter of opinion, I prefer GitHub's "hub" instead of that setup.




> why would it stay in IPython/IPython ?

I'd be happy if it was moved out to Jupyter/backport-pr or something similar